.github/workflows/changeset.yml

```diff
@@ -1,58 +1,58 @@
 name: Changesets
```

Comment:
<img width="1073" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/b497beda-7468-454a-8f62-f0190af1373d">

There's a way to unselect all radios, is this intended as radio can't be "unselected" usually.

Reply:
<img width="240" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/31df7fd6-bc96-42bb-a32b-8ee1d8895241">
Do we want to be able to select the titles?

Reply:
<img width="466" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/1b5add92-8450-4363-beeb-c271adb5a07b">
This font size seems odd.

Reply:
1. It does seem a bit weird, I could change the selection behavior here, but honestly it's up to the user how it's used. I could however, put the default selection behavior for radios to be not deselectable.
2. Section names are not selectable.
3. Well I didn't have a design for this part. Should it follow the same font as the items?

Reply:
3. That's my main point, it looks different from the Louise and Karen
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/22615a81-0aa1-400b-89ae-502febf39143)